File: beginner/javascript/2159.js
var fs = require("fs");
var input = fs.readFileSync("stdin", "utf8");
var lines = input.split("\n");

const n = parseInt(lines.shift());
  
const result1 = n / Math.log(n);
const result2 = 1.25506 * (n / Math.log(n));

console.log(result1.toFixed(1), result2.toFixed(1));

File: beginner/java/2159.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.close();
        
        double result1 = n / Math.log(n);
        double result2 = 1.25506 * n / Math.log(n);
        
        System.out.printf("%.1f %.1f%n", result1, result2);
    }
}